### ISSUE TITLE
Add sample for use with Binary dataset

### ISSUE THREAD

**Issue by PeterPann23** [dotnet/machinelearning#3748]
[Enter feedback here]

Would be nice to show how a binary file can be used (and re-used) to use strong typed data
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d77f7787-449e-9eeb-9dc8-e7ca22091538
* Version Independent ID: 2dca3efe-4be8-72c3-0e71-226c2183cc89
* Content: [DataOperationsCatalog.CreateEnumerable(IDataView, Boolean, Boolean, SchemaDefinition) Method (Microsoft.ML)](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.dataoperationscatalog.createenumerable?f1url=https%3A%2F%2Fmsdn.microsoft.com%2Fquery%2Fdev15.query%3FappId%3DDev15IDEF1%26l%3DEN-US%26k%3Dk(Microsoft.ML.DataOperationsCatalog.CreateEnumerable%60%601);k(SolutionItemsProject);k(DevLang-csharp)%26rd%3Dtrue&view=ml-dotnet)
* Content Source: [dotnet/xml/Microsoft.ML/DataOperationsCatalog.xml](https://github.com/dotnet/ml-api-docs/blob/live/dotnet/xml/Microsoft.ML/DataOperationsCatalog.xml)
* Product: **dotnet-ml-api**
* GitHub Login: @sfilipi
* Microsoft Alias: **johalex**

**Comment by CESARDELATORRE** [comment]
@PeterPann23  - Can you clarify what do you mean with a 'binary file'?
Do you mean an IDV dataset? (.IDV is the IDataView proprietary serialization format).

Or do you mean a different kind of binary file?

Thanks,
Cesar

**Comment by PeterPann23** [comment]
Yes, yes I do, there is a convention to call/ save it using .idv extension, however 
this suggestion is not written anywhere in the the code comments and the methods are called  
mlContext.Data.SaveAsBinary(dataView, fileName);
mlContext.Data.LoadFromBinary(fileName);

so to call it binary or to call "Internal Data View" or what does it stand for...

**Comment by CESARDELATORRE** [comment]
@PeterPann23 - I agree. This should also be in the docs.microsoft.com pages.
I just created another issue in there related to this:

https://github.com/dotnet/docs/issues/12451

**Comment by najeeb-kazmi** [comment]
@CESARDELATORRE  @PeterPann23 is this related to #3661 ? I have a PR #3745 out that adds a sample for Binary load/save, as well as Text load/save.

**Comment by CESARDELATORRE** [comment]
@najeeb-kazmi - Yep! Completely related! That PR is great to have.
But we should also document it in a more "tutorial" approach at docs.microsoft.com so that's why I opened another issue at docs.microsoft.com

Thanks Najeeb! 👍

**Comment by najeeb-kazmi** [comment]
@CESARDELATORRE @PeterPann23 - Shall we close this issue then, since it's covered by the PR on the ML.NET side, and we have another issue to track the tutorial on docs.microsoft.com?
